### PR TITLE
Nh/lint/grid view example

### PR DESCRIPTION
**Pull request by nhachicha**

### CODE REVIEW

**Comment by zaki50** [comment]
I got `PrngFix` warning.

please add

```
<issue id="PrngFix" severity="ignore" />
```
to `lint.xml`.

**Comment by zaki50** [comment]
:+1:

File: examples/gridViewExample/lint.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <!-- Disable the given check in this project -->
+    <issue id="AllowBackup" severity="ignore" />
+    <issue id="IconLauncherShape" severity="ignore" />
+    <issue id="IconMissingDensityFolder" severity="ignore" />
+    <issue id="GoogleAppIndexingWarning" severity="ignore" />
+
```
Comment (cmelchior) [inline examples/gridViewExample/lint.xml]: Remove whitespace

File: examples/gridViewExample/src/main/res/layout/activity_realm_example.xml [hunk 2]
```diff
@@ -13,15 +13,15 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:gravity="center_horizontal"
-        android:text="My Favorite City"
+        android:text="@string/my_favortie_city"
```
Comment (cmelchior) [inline examples/gridViewExample/src/main/res/layout/activity_realm_example.xml]: favortie -> favorite

**Pull request merged**